build-configs/ThemeType.ts

```diff
@@ -1,7 +1,40 @@
-import { ColorsType } from './common/theme/colors'
-import { FontsType } from './common/theme/fonts'
+export type SimplePaletteColor = {
```

Comment:
I decided to recreate the types myself to enable cross platform compatibility (as mui is not available for native). I am not totally convinced this is the right way to go, but seemed like the best approach for now.

Reply:
True, but installing that library does not work/does not make sense for web (and vice versa with mui). This is a shared workspace so installing a web only/native only solution does not work.